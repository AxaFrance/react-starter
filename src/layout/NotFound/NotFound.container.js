import React, { useEffect } from 'react';
import { NotFound } from './NotFound';

const EnhancedNotFound = () => {
  useEffect(() => {}, []);

  return <NotFound />;
};
export default EnhancedNotFound;
