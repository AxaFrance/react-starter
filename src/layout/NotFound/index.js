export { NotFound } from './NotFound';
export { default } from './NotFound.container';
