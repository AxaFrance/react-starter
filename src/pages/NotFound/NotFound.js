import React from 'react';
import './NotFound.scss';

const NotFound = () => (
  <>
    <div className="container af-container--notfound">
      <h1 className="af-notfound__title">
        <div className="af-notfound__title-covernumber">
          <span className="af-notfound__title-number">404</span>
        </div>
        <div className="af-notfound__title-covernot">
          <span className="af-notfound__title-not">
            not
            <br />
            found
          </span>
        </div>
      </h1>
      <p className="af-notfound__message">
        The page you are looking for is not here!
      </p>
    </div>
  </>
);

export default NotFound;
