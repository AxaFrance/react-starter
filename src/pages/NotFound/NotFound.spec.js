import React from 'react';
import { render } from '@testing-library/react';
import NotFound from './NotFound';

it('1. Renders NotFound component without crashing', () => {
  const { asFragment } = render(<NotFound title="test" />);
  expect(asFragment()).toMatchSnapshot();
});
