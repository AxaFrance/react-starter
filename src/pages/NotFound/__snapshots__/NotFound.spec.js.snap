// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`1. Renders NotFound component without crashing 1`] = `
<DocumentFragment>
  <div
    class="container af-container--notfound"
  >
    <h1
      class="af-notfound__title"
    >
      <div
        class="af-notfound__title-covernumber"
      >
        <span
          class="af-notfound__title-number"
        >
          404
        </span>
      </div>
      <div
        class="af-notfound__title-covernot"
      >
        <span
          class="af-notfound__title-not"
        >
          not
          <br />
          found
        </span>
      </div>
    </h1>
    <p
      class="af-notfound__message"
    >
      The page you are looking for is not here!
    </p>
  </div>
</DocumentFragment>
`;
